maas-schemas/schemas/core/components/car-rental.json

```diff
@@ -97,8 +114,7 @@
           "description": "Current geo location",
           "allOf": [{ "$ref": "http://maasglobal.com/core/components/units-geo.json#/definitions/location" }]
         }
-      },
-      "required": ["classification"]
```

Comment:
This change seems unrelated. Was this an accident or was it intentional?

Reply:
It is intentional. The PR was initially just for this change. However the change to try fixing Maui is also added by me later, because I might as well fix two in one go